pkg/agent/controller/networkpolicy/reconciler_test.go

```diff
@@ -1810,3 +1919,74 @@ func BenchmarkGroupPodsByServicesWithNamedPort(b *testing.B) {
 func BenchmarkGroupPodsByServicesWithoutNamedPort(b *testing.B) {
 	benchmarkGroupMembersByServices(b, false)
 }
+
+// policyRuleMatcher implements gomock.Matcher.
+// It is used to check whether the argument of the mocked method is expected. It ignores the difference of item order of
+// slices and some fields including "Priority" and "FlowID" which are a little difficult to predict.
```

Comment:
It ignores differences in slice element order and some fields...

Reply:
done, thanks